### ISSUE TITLE
custom entries in .syscommand missing after upgrade

### ISSUE THREAD

**Issue by pmbert** [Z-Wave-Me/home-automation#487]
After upgrade to Z-Way version v2.3.7 and v2.3.8 all custom entries in .syscommands are missing and must be newly set.

**Comment by PoltoS** [comment]
Yes, this is a know problem. Not solved yet. You can find your file in the neightbouring folder.
